src/app/content/components/Attribution.tsx

```diff
@@ -64,26 +54,18 @@ const Content = styled.div`
 `;
 
 // tslint:disable-next-line:variable-name
-const Details = styled.details`
+const AttributionDetails = styled(Details)`
```

Comment:
Why wouldn't this use the `DetailsComponent` ?

Reply:
the DetailsComponent in sidebar only exists to provide the `defaultOpen` functionality so that when you nav around the changing props don't make the details re-render themselves and collapse when they shouldn't

i suppose it wouldn't _hurt_ to use it everywhere, but its also extra logic that nothing else needs